### PR TITLE
New function and bug fixes

### PR DESCRIPTION
**Pull request by SeonerVorteX**
# Changes
## Added build system
- added **build** script to `package.json`
- added **tsc-alias** module for building (as dev dependency)
- added **/dist** to `.gitignore`
- added **NODE_ENV** to `.env` to separate production and development
- - `NODE_ENV="development"` in development
- - `NODE_ENV="production"` in production
- **start** script changed
## Bug fixes and new functions
- **SQLHelper.ts** file has been deleted due to no benefit
- `No such a file or directory` error on logs have been fixed
- - The problem is that Windows doesn't support ":" in file name
- logs file name changed **YYYY:MM:DD** => **YYYY-MM-DD**
- **Invalid Birthday** issue has been resolved 
- created a new safe function to create file: **$writeToFileSafe**
- `tsconfig.json` changed
- - comment line deleted
- - **noEmit** options deleted, because it is blocking to build to `/dist/` folder
- - added new lib **DOM** to resolve the **Cannot find name "console"** issue

### CODE REVIEW

File: src/assets/constants/general.ts [hunk 1]
```diff
@@ -9,5 +14,23 @@ export const ASCII_logo = `|||||||||||||||||||||||||||||||||||||||||||||||||||||
 ||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
 `;
 
-export const currentLogFilePath = 'src/bin/logs/logs.txt';
-export const logsBasePath = 'src/bin/logs';
+const isDev = process.env.NODE_ENV === 'development';
```
Comment (vugarsafarzada) [inline src/assets/constants/general.ts:17]: NODE_ENV yə ehtiyac yoxdu DEVELOPER_MODE = true/false dəyərlərinə görə dəyişə bilərsən?

File: src/assets/helpers/logHelpers.ts [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
 import { sendLogToTelegramBot } from "#helpers/TelegramBot";
 import dotenv from "dotenv";
+import { $writeToFileSafe } from "./methods";
 dotenv.config();
 
 export const initLogs = () => {
     let beforeLogs = readFromFile(currentLogFilePath) || '-';
-    writeToFile(beforeLogs, `${logsBasePath}/logs-${moment().format('DD.MM.YYYY:HH:mm:ss')}.logs.txt`);
-    writeToFile(ASCII_logo, currentLogFilePath);
+    $writeToFileSafe(beforeLogs, `${logsBasePath}/logs-${moment().format('DD.MM.YYYY-HH-mm-ss')}.logs.txt`); // Windows doesn't support ':' in file name
```
Comment (vugarsafarzada) [inline src/assets/helpers/logHelpers.ts:12]: bu file dakı bütün (.txt -> .log) et

File: tsconfig.json [hunk 3]
```diff
@@ -85,12 +83,11 @@
     }
   },
   "include": [
-    "./src/**/*",
-    "./src"
+    "src"
```
Comment (vugarsafarzada) [inline tsconfig.json:86]: Bunu niyə dəyişdin ki

**Review by vugarsafarzada — CHANGES_REQUESTED** [review]

File: tsconfig.json (hunk 3 above)
Comment (SeonerVorteX) [reply tsconfig.json:86]: Çünki eyni 2sinin qısaltmasıdı, eyni şeyi edir

File: src/assets/helpers/logHelpers.ts (hunk 2 above)
Comment (SeonerVorteX) [reply src/assets/helpers/logHelpers.ts:12]: okay

**Review by SeonerVorteX — COMMENTED** [review]
I've done all of your requests

**Review by vugarsafarzada — APPROVED** [review]

**Pull request merged**